plugins/modules/aci_l3out_logical_node.py

```diff
@@ -47,8 +47,8 @@
     description:
     - Configure the router ID as a loopback IP.
     type: str
-    choices: [ 'yes', 'no' ]
-    default: 'yes'
+    choices: [ true, false]
```

Comment:
Will this module still work? The argument_spec still requires `router_id_as_loopback=dict(type="str", default="yes", choices=["yes", "no"])`, so would expect the code to be adjusted to accept bool input and map to string that should be configured on APIC.

Reply:
changed for yes/no